src/client/common/components/empty-network.js

```diff
@@ -7,14 +7,16 @@ const { Link } = require('react-router-dom');
 
 class EmptyNetwork extends React.Component {
   render(){
-    let { msg } = this.props;
+    let { msg, app} = this.props;
+
+    let linkToPCSearch = (app === 'interactions') ? ['Return to', h(Link, { className: 'plain-link', to: { pathname: '/' } },  ' Pathway Commons Search')] : null;
```

Comment:
Instead of passing a string you should pass a boolean prop that determines whether to show the return to pc link or not.

Reply:
i.e. make the prop more general instead of specifically an app string.

In EmptyNetwork:
```js
let { msg, showPcLink } = this.props;

let linkToPCSearch = showPcLink ? ...
```

in enrichment/index.js:
```js

h(EmptyNetwork, { msg: ..., showPcLink: true } )
```

Reply:
Sorry, I overlooked this comment earlier.